studio-docs/source/org/graphs.mdx

```diff
@@ -1,28 +1,64 @@
 ---
-title: Managing graphs and variants
+title: Graphs and variants
 sidebar_title: Graphs and variants
 ---
 
-A **graph** in Apollo Studio represents the entirety of a connected data graph. Every graph has its own associated GraphQL schema.
+A **graph** in Apollo Studio represents a connected data graph in your organization. Each graph has one or more **variants**, which correspond to the different environments where the graph runs (such as staging and production). Each variant has its own GraphQL schema, which means schemas can differ between environments.
 
 ## Creating a graph
 
-To create a graph in Studio, first select the Studio organization that the graph will belong to. Then click **New Graph** in the upper right and proceed through the creation flow.
+To create a graph in [Apollo Studio](https://studio.apollographql.com/), first select the Studio organization that the graph will belong to. Then click **New Graph** in the upper right and proceed through the creation flow:
 
-Note that every graph in Studio has a globally unique **graph name**. We recommend that you prefix your graph names with the name of your company or organization to avoid naming collisions.
+<img class="screenshot" src="../img/graph-creation.jpg" width="500" />
+
+There are two types of graphs in Studio:
+
+* **Deployed graphs** are shared with other members of your organization. Create a deployed graph for every use case _except_ local development.
+* **Development graphs** (**dev graphs** for short) are only visible to you. Use them to help you iterate on your graph in your development environment.
+
+Note that every graph in Studio has a **globally unique name**. We recommend that you prefix your graph names with the name of your organization to avoid naming collisions.
```

Comment:
We might be able to take this out for now since we've removed the ability for you to set your own slug (temporarily) and will just set one for you based on the title you give us that avoids collisions.